src/static/lookup.php

```diff
@@ -45,7 +45,8 @@
 
 <?php
 
-
+require_once "config.php";
+  maintable = 
 $user = filter_input(INPUT_POST, 'user_id');
 
 $servername = "cs-3250-database-1testing.ctxpxr8jzoap.us-west-1.rds.amazonaws.com";
```

Comment:
this connection to the db should be deleted now that the config.php is connected immediately above